awx/ui_next/src/screens/Inventory/InventoryHosts/InventoryHosts.jsx

```diff
@@ -1,46 +1,30 @@
 import React from 'react';
-import { Switch, Route, withRouter } from 'react-router-dom';
+import { Switch, Route } from 'react-router-dom';
 
 import Host from '../../Host/Host';
 import InventoryHostList from './InventoryHostList';
-import InventoryHostAdd from '../InventoryHostAdd';
+import HostAdd from '../../Host/HostAdd';
 
-function InventoryHosts({ match, setBreadcrumb, i18n, inventory }) {
+function InventoryHosts({ setBreadcrumb, inventory }) {
   return (
     <Switch>
-      <Route
-        key="host-add"
-        path="/inventories/inventory/:id/hosts/add"
-        render={() => <InventoryHostAdd match={match} />}
-      />
-      ,
+      <Route key="host-add" path="/inventories/inventory/:id/hosts/add">
```

Comment:
I think we should keep our routing pattern consistent throughout the app and it's something to keep an eye on as we continue to develop as it's probably out of scope for this PR.

Reply:
Yes, I completely agree that we should be consistent. I debated between whether or not I should update all of the routes, but decided against it - I didn't want to overwhelm the reviewers with a huge diff not related to the issue.